index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Prince Resume</title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="styles.css">
+
+</head>
+
+<body>
+
+  <hr>
+  <h1>Prince Rogers Nelson</h1>
+  <h2> June 7, 1958 - 4ever</h2>
+  <h3>The <span> Inimitable </span> Embodiment of Music</h3>
+
+  <nav>
+        <strong>About Prince</strong>
+        &#9829;
+        <a href="#">Movies</a>
+        &#9829;
+        <a href="http://princevault.com/index.php?title=A-Z_song_list" target="_blank">List of Songs</a>
+  </nav>
+
+  <img class="super" src="images/prince superbowl.jpeg">
+
+  <hr>
+
+      <p> I am Prince. I came to share the gift of music and to try to enrich people's lives, whether by giving them a soundtrack to reference the span of their journey or just providing enjoyment in a solitary moment of time. I held myself to the highest standards and strove to only deliver my best. If I didn't feel a song was an accurate display of my talent/message and completed to the point where the creation thoroughly matched the inspiration, I wouldn't share it...which is the reason for my vault of music. As those items go public, just know they were created out of love. Love for you, love for myself and love of creating something tangible out of the intangible. I lived my life by my own rules and once the "powers that be" could no longer tolerate that, they sought to end me...<span> but I am eternal</span>. If you ever feel I am gone, simply play one of my songs...and there I will be.
+      </p>
+
+    <div class="pianopics">
+
+    <img src="images/princeearlypiano.jpg" style="float: left; width: 40%;> "><img src="images/princepianostanding.jpeg" stlye= "float: right"; height="102%"; width="60%"> 
```

Comment:
@agreen2017 the reason the image is not loading is because you are requesting `src="images/princeearlypiano.jpg"` , however that image is inside of folder called `Image` not `image`. In the future try to name all your files and folders using all lowercase to avoid these issues. 🔨